src/components/oscal-utils/OSCALBackMatterUtils.js

```diff
@@ -19,28 +19,41 @@ export function getAbsoluteUrl(rlink, parentUrl) {
   return absoluteUrl;
 }
 
+/**
+ * Gets a resource from the given back matter's resource by the given HREF UUID.
+ */
+export function getResourceFromBackMatterByHref(backMatter, href) {
+  if (!href.startsWith("#")) {
+    throw new Error("not an internal href");
```

Comment:
I'm not sure we want to display the href in the UI just from a usability perspective.

I'd be fine with logging as long as we can keep it simple in this iteration and not introduce new logging frameworks.

Reply:
Linting is warning about direct use of console too (and for decent reason), so a more comprehensive probably is warranted and I'm tempted to kick this can down the road a bit.